lms/djangoapps/branding/api.py

```diff
@@ -451,7 +451,9 @@ def _footer_logo_img(is_secure):
     Returns:
         Absolute url to logo
     """
-    logo_name = configuration_helpers.get_value('FOOTER_ORGANIZATION_IMAGE', settings.FOOTER_ORGANIZATION_IMAGE)
+    default_local_path = 'images/logo.png'
+    footer_image_url = settings.LOGO_TRADEMARK_URL
+    footer_logo_path = configuration_helpers.get_value('FOOTER_ORGANIZATION_IMAGE', settings.FOOTER_ORGANIZATION_IMAGE)
```

Comment:
Note:
We should skip using `FOOTER_ORGANIZATION_IMAGE` and unset its value as we are moving ahead. 
https://github.com/edx/edx-internal/search?q=FOOTER_ORGANIZATION_IMAGE

Reply:
Yes, we can remove `FOOTER_ORGANIZATION_IMAGE` to simplify the code to use only the `LOGO_TRADEMARK_URL`. We should follow-up by creating a DEPR ticket to inform the community.

If for any reason, we need to re-add `FOOTER_ORGANIZATION_IMAGE`, we can do the following:
* Set `FOOTER_ORGANIZATION_IMAGE` to `None` for edx.org's configuration.
* The preference order would be: `FOOTER_ORGANIZATION_IMAGE` if exists, otherwise `LOGO_TRADEMARK_URL`.